### PR TITLE
Add RabbitMQ

### PR DESCRIPTION
**Pull request by dcole-inviqa**

### CODE REVIEW

File: rabbitmq/docker-compose.sample.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  rabbitmq:
+    image: "rabbitmq:3-management"
```
Comment (kierenevans) [inline rabbitmq/docker-compose.sample.yml]: This would need to be `quay.io/continuouspipe/rabbitmq36-management:stable`

File: docker-compose.yml [hunk 2]
```diff
@@ -559,6 +559,11 @@ services:
     depends_on:
       - ubuntu
 
+  rabbitmq36_management:
+    build:
+      context: ./rabbitmq/
+    image: quay.io/continuouspipe/rabbitmq36-management:latest
```
Comment (kierenevans) [inline docker-compose.yml]: Is it possible to declare an external_rabbitmq36_management that this depends on too? Not sure we still use the build.sh script which pulls any external_* beforehand, but might be useful still.

**Comment by kierenevans** [comment]
Looking good, we should get a README in here too :)

File: docker-compose.yml (hunk 2 above)
Comment (dcole-inviqa) [reply docker-compose.yml]: added external dependency

**Pull request merged**